plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java

```diff
@@ -233,6 +239,29 @@ public void onEnable() {
             return;
         }
 
+        try {
+            this.adventure = BukkitAudiences.create(this);
+        }
+        catch (Exception exception) {
+            logger.error("Could not initialize adventure platform", exception);
+            this.shutdown("Critical error has been encountered!");
+            return;
+        }
+        this.localeManager = new LocaleManager();
+        try {
+            Result<File, String> createResult = FunnyIOUtils.createFile(this.pluginLanguageFolderFile, true);
+            if (createResult.isErr()) {
+                this.shutdown(createResult.getError());
+                return;
+            }
+            this.messageService = this.prepareMessageService();
+        }
+        catch (Exception exception) {
+            logger.error("Could not initialize message service", exception);
+            this.shutdown("Critical error has been encountered!");
+            return;
+        }
```

Comment:
to przydałoby się wydzielić do innej klasy, łącznie z prepareMessageService() jest sporo tutaj takiej internal logiki od ładowania plików